website/templates/base.html

```diff
@@ -56,9 +56,9 @@
                                 {% if request.user.userprofile.avatar %}
                                     <img src="{{ request.user.userprofile.avatar }}">
                                 {% elif request.user.socialaccount_set.all.0.get_avatar_url %}
-                                     <img src="{{request.user.socialaccount_set.all.0.get_avatar_url}}">
+                                    <img src="{{request.user.socialaccount_set.all.0.get_avatar_url}}">
                                 {% else %}
-                                    <img src="{% gravatar request.user.email 20 %}">
+                                    {% gravatar request.user.email 20 %}
```

Comment:
This is not the right fix. 
Please put this line: `<img src="{% gravatar_url request.user.email 20 %}">` instead of `{% gravatar request.user.email 20 %}`